src/lib/intent-validator/tag-update.ts

```diff
@@ -0,0 +1,20 @@
+import SchemaValidator from './schema-validator'
+import * as Joi from 'joi'
+
+class TagUpdateStepValidator extends SchemaValidator {
+  // TODO: Find a way to remove these protected properties as they are not used here.
```

Comment:
What do you mean? The base class uses those properties to render the validation errors.

https://github.com/contentful/contentful-migration/blob/master/src/lib/intent-validator/schema-validator.ts#L34-L35

Reply:
Ah, that was my confusion. I thought these were properties only relevant to content types and overlooked that they are about how to render the validation errors. I will remove the comment.